pkg/kubepkg/kubepkg.go

```diff
@@ -145,13 +181,23 @@ func ConstructBuilds(packages, channels []string, kubeVersion, revision, cniVers
 	return builds, nil
 }
 
-func WalkBuilds(builds []Build, architectures []string) error {
+func WalkBuilds(builds []Build, architectures []string, specOnly bool) error {
 	logrus.Infof("Walking builds...")
 
+	tmpDir, err := ioutil.TempDir(os.TempDir(), "kubepkg")
+	if err != nil {
+		return err
+	}
+
+	_, err = os.Stat(tmpDir)
+	if err != nil {
+		return err
+	}
```

Comment:
I think we can remove this because `ioutil.Tempdir()` should ensure that the dir exists.